### ISSUE TITLE
What does args.ood_coeff and loss_ood mean?

### ISSUE THREAD

**Issue by awei-97** [naver-ai/w-ood#7]
Sorry , It's a little hard for me to understand the code about the calculation on  **loss_ood** .  

What does **args.ood_coeff** and **loss_ood mean**? 

Is the explaination on **loss_ood** writing in the article ?

  Can you explain it or offer some link to explain where does it come from?

And thanks a lot.

Specificly，about these codes :
`      if args.ood_coeff > 0:
                try:
                    pack_ood = next(train_data_loader_ood_iter)
                except:
                    train_data_loader_ood_iter = iter(train_data_loader_ood)
                    pack_ood = next(train_data_loader_ood_iter)
                img_ood = pack_ood['img'].cuda()

                label_ood = torch.zeros([args.ood_batch_size, 20]).cuda()

                for adv_iter in range(1):
                    img_ood.requires_grad = True
                    logit_ood, out_features_ood = model.feature_list(img_ood)
                    logit_max, _ = torch.max(logit_ood, dim=1)

                    loss_adv = logit_max.sum()
                    model.zero_grad()
                    loss_adv.backward()
                    data_grad = img_ood.grad.data
                    perturbed_data = adv_climb(img_ood, 0.07, data_grad).detach()
                    img_ood = perturbed_data
                logit_ood_perturb, out_features_ood_perturb = model.feature_list(perturbed_data)
                logit_max_perturb, logit_max_perturb_idx = torch.max(logit_ood_perturb, dim=1)

                loss_mask = (logit_max_perturb_idx != 4)
                loss_ood = F.multilabel_soft_margin_loss(logit_ood_perturb, label_ood, reduce=False)`
